kernels/volk/volk_8u_conv_k7_r2puppet_8u.h

```diff
@@ -188,87 +188,87 @@ static inline void volk_8u_conv_k7_r2puppet_8u_spiral(unsigned char* syms,
 #endif /*LV_HAVE_SSE3*/
 
 
-#if LV_HAVE_AVX2
-
-#include <immintrin.h>
-#include <stdio.h>
-
-static inline void volk_8u_conv_k7_r2puppet_8u_avx2(unsigned char* syms,
-                                                    unsigned char* dec,
-                                                    unsigned int framebits)
-{
-
-
-    static int once = 1;
```

Comment:
Might that be a reason it works in our CI but not in applications?

Reply:
If I remember correctly the puppet function are only used in testing.